frontend/src/modules/EditZing/Components/StudentCard.tsx

```diff
@@ -121,6 +183,52 @@ const StudentCard = ({
                 top: '1px',
               }}
             />
+
+            <IconButton
+              component="button"
+              sx={{
+                border: 'none',
+                width: '24px',
+                height: '24px',
+                position: 'absolute',
+                right: '1px',
+                bottom: '4px',
+                cursor: 'pointer',
+                display: !studentNotes && isHovering ? '' : 'none',
+              }}
+              onClick={handleOpenNotes}
+              color="secondary"
+            >
+              <img
+                src={notesIcon}
+                style={{ width: '24px', height: '24px' }}
+                alt="edit student note"
+              />
```

Comment:
Instead of using images, look into using an [SvgIcon](https://mui.com/material-ui/icons/#svgicon) component.

Reply:
Yeah idrk how to use svgs. Should I first convert the image into a svg and then put the path and stuff as the children of the SvgIcon?

Reply:
yep, that's right. if you downloaded it from Figma, I think you can download it directly as an SVG. You can either copy and paste the path directly or import it as a component.